hyperspace/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM rustlang/rust:nightly-buster as backend-builder
+
+WORKDIR /usr/src/hyperspace-build
+COPY . .
+
+# Protobuff issue with version 3 and optional features
+# Installing a specific version that works locally for Blas
+RUN curl -LO https://github.com/protocolbuffers/protobuf/releases/download/v3.12.4/protoc-3.12.4-linux-x86_64.zip
+
+RUN unzip protoc-3.12.4-linux-x86_64.zip -d .local
+ENV PATH="${PATH}:/usr/src/hyperspace-build/.local/bin"
+RUN protoc --version
+
+RUN make build-release-hyperspace
+RUN ls -l ./target/release
+
+FROM debian:buster-slim
+
+COPY --from=backend-builder /usr/src/hyperspace-build/target/release/hyperspace .
+CMD ["./hyperspace"]
```

Comment:
@obsessed-cake any hints on how to fix this?

Reply:
docker run ${IMAGE} ${ARG} will completely substitute what is in the `CMD` instruction for ${ARG}, so here you are effectively calling `relay`, which matches the actual error: `exec: "relay": executable file not found in $PATH: unknown.`

Reply:
You should use `ENTRYPOINT` instead of `CMD` here.
I fixed it:
```
$ docker run --rm -ti hyperspace:latest relay --help
hyperspace-relay 
Start relaying messages between two chains

USAGE:
    hyperspace relay [OPTIONS] --config <CONFIG>

OPTIONS:
        --config <CONFIG>                Relayer config path
        --delay-period <DELAY_PERIOD>    Connection delay period in seconds
    -h, --help                           Print help information
        --order <ORDER>                  Channel order
        --port-id <PORT_ID>              Port id for channel creation
        --version <VERSION>              Channel version

$ docker run --rm -ti -v$(pwd)/hyperspace/config.toml:/tmp/config.toml  hyperspace:latest relay --config /tmp/config.toml
Error: Rpc Error Transport(Error when opening the TCP socket: Connection refused (os error 111))

```